.gitignore

```diff
@@ -31,6 +31,8 @@ node_modules/
 
 # ???? consider tracking the entire plugins directory
 public/wp-content/cache/*
+public/wp-content/plugins/*
+
 public/wp-content/plugins/sg-cachepress/*
 public/wp-content/plugins/wordpress-starter/*
```

Comment:
This can be removed.

Reply:
![image](https://user-images.githubusercontent.com/38733594/79267679-92550c80-7e67-11ea-8734-23198b680065.png)

is this what you wanted?

Reply:
If you're ignoring all plugins, you don't need to block extra plugins by name. Please fix.